agent/src/main/scala/za/co/absa/atum/agent/exception/AtumAgentException.scala

```diff
@@ -16,15 +16,35 @@
 
 package za.co.absa.atum.agent.exception
 
+/**
+ * This class represents an exception thrown by the Atum Agent.
```

Comment:
```suggestion
 * This class represents a basis for exceptions thrown by the Atum Agent.
```
?

Reply:
or something similar

Reply:
Change to "This type represents a base class for exceptions thrown by the Atum Agent." which is close to your suggestion emphasizing it's a basis for other exceptions.